index.js

```diff
@@ -1,4 +1,6 @@
 (function() {
+  var isBrowser = (typeof window !== 'undefined');
+  var React = isBrowser ? window.React || require('react') : null;
```

Comment:
don't use `null` in JS, there are no pointers and it tends to almost always just get coerced to `false`, so let's just make this `false` from the get-go.